requirements.txt

```diff
@@ -3,7 +3,8 @@ click>=6.2
 Jinja2>=3.1
 PyYAML>=3.1
 docker-py>=1.6.0
-requests>2.11
+requests<=2.5.3
```

Comment:
How is this change related to this PR?

Reply:
Definitely need to turn this into a separate PR. Thanks!

Reply:
I was encountering errors while using the latest version of requests: https://github.com/cf-platform-eng/tile-generator/pull/332